src/EPPlus/FormulaParsing/ErrorValues/ExcelErrorValue.cs

```diff
@@ -303,7 +227,7 @@ public override string ToString()
         /// <returns></returns>
         public override int GetHashCode()
         {
-            return (int)Type;
+            return base.GetHashCode();
```

Comment:
We can discuss this next week, there were no tests failing because of this change from what I can see. To use the `Type` as hashcode will not work so well with the cell images since the object that is stored in the `cell.Value` is inheriting `ExcelErrorValue` (they all are stored as a #VALUE error and an image) but should not be compared as the same instance even if they have the same `Type` value.